templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -51,10 +53,48 @@
     "# os.environ[\"WANDB_API_KEY\"]=\"YOUR_WANDB_API_KEY\"\n",
     "\n",
     "# Launch a LoRA fine-tuning job for Llama 3 8B with 16 A10s\n",
-    "!python main.py training_configs/lora/llama-3-8b.yaml\n",
+    "!llmforge anyscale finetune training_configs/lora/llama-3-8b.yaml\n",
     "\n",
     "# Launch a full-param fine-tuning job for Llama 3 8B with 16 A10s\n",
-    "# !python main.py training_configs/full_param/llama-3-8b.yaml"
+    "# !llmforge anyscale finetune  training_configs/full_param/llama-3-8b.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`LLMForge` is an Anyscale specific CLI and library that is installed on this workspace so that you can quickly experiment and customize various LLM finetuning experiments by simply modifying a config file. For extensive documentation around what is supported through the config refer to [docs](https://docs.anyscale.com/reference/finetuning-config-api/). "
```

Comment:
```suggestion
    "`LLMForge` is an Anyscale CLI and library that is installed on this workspace so that you can quickly experiment and customize various LLM finetuning experiments by simply modifying a config file. For extensive documentation around what is supported through the config refer to [docs](https://docs.anyscale.com/reference/finetuning-config-api/). "
```